packages/fvm/src/address/index.ts

```diff
@@ -1,9 +1,6 @@
 // Forked & converted to use `bigint` & `Uint8Array`
 // Source: https://github.com/Zondax/izari-filecoin/blob/master/src/address/index.ts
-import { compare as u8aCompare } from "uint8arrays/compare";
-import { concat as u8aConcat } from "uint8arrays/concat";
-import { fromString as u8aFromString } from "uint8arrays/from-string";
-import { toString as u8aToString } from "uint8arrays/to-string";
+import { Hex, bytesToHex, concatBytes, hexToBytes } from "viem";
```

Comment:
we replaced the `fvm` package's `uint8arrays` dep with `viem`, which has similar bytes<>hex utilities that we need. this allows the package to work on the web.